### PR TITLE
fix: update crossreference markup to new autorefs format

### LINKED ISSUE #6: bug: deprecation warning with latest mkdocstrings and autorefs
### Description of the bug

The following deprecation warning gets output with the latest mkdocstrings version:

https://github.com/mkdocstrings/autorefs/blob/40940c591e9cc557573ccd35351ca2b535ec371b/src/mkdocs_autorefs/references.py#L274-L279

I am aware that this is the legacy Python handler. Despite that, is there still interest in having this fixed?

### PR DESCRIPTION
**Pull request by mschoettle**
Fixes #6

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
-    "mkdocstrings>=0.19",
+    "mkdocstrings>=0.26",
```
Comment (mschoettle) [inline pyproject.toml:31]: Not sure if this is necessary. Changed this since `mkdocstrings` 0.26.0 starts requiring autorefs v1+.

Comment (pawamoy) [reply pyproject.toml:31]: Let's depend on mkdocs-autorefs v1 directly then.

Comment (pawamoy) [reply pyproject.toml]: Actually, the `autoref` element was introduced in v1.1. I've pushed to your branch :slightly_smiling_face:

**Comment by pawamoy** [comment]
Thanks a lot!

**Pull request merged**